Libraries/Components/TextInput/TextInput.js

```diff
@@ -1601,6 +1602,14 @@ const ExportedForwardRef: React.AbstractComponent<
     React.ElementRef<HostComponent<mixed>> & ImperativeMethods,
   >,
 ) {
+  let style = flattenStyle(restProps.style);
+
+  if (style && style.verticalAlign !== undefined) {
+    // $FlowFixMe
```

Comment:
Q: Why you need this fix me, flow is not showing any errors for me.

Reply:
Yeah, that's a bit weird but when I originally opened this PR there was a flow error about us mutating the style object but I guess with these new updates (maybe the new flow version) that's no longer a problem.